### PR TITLE
[#127] Made dynamic condition for no managed files step.

### LINKED ISSUE #127: Inconsistent format of "Given no managed files:" step
The step's help currently states the following usage:
```
Given no managed files:
| filename      |
| myfile.jpg    |
| otherfile.jpg |
```

The code, however, does not account for the name of the provided field `filename` and is hardcoded to `filename` instead of using a field name. 
Moreover, the `| filename      |` is removed in the step.

Actions:
1. Remove hardcoded `filename` and use the provided field name.
2. Updated step comment to reflect that any field name can be specified in the column name.

### PR DESCRIPTION
**Pull request by tannguyen04**
closes #127

### CODE REVIEW

File: src/FileTrait.php [hunk 1]
```diff
@@ -136,11 +136,14 @@ public function fileCleanAll(AfterScenarioScope $scope): void {
   }
 
   /**
-   * Delete managed files defined by provided properties.
+   * Delete managed files defined by provided properties/fields.
+   *
+   * Example: filename, uri, status, uid and more see Drupal\file\Entity\File.
```
Comment (AlexSkrypnyk) [inline src/FileTrait.php:141]:
```
Example: filename, uri, status, uid and more.

@see Drupal\file\Entity\File
```

File: src/FileTrait.php [hunk 2]
```diff
@@ -136,11 +136,14 @@ public function fileCleanAll(AfterScenarioScope $scope): void {
...
+   *
    *
    * @code
    * Given no managed files:
-   * | filename      |
+   * | fieldName     |
```
Comment (AlexSkrypnyk) [inline src/FileTrait.php:146]: please revert this change and add a second `@code/@endcode` with another example of using a different field name - `path`.

File: src/FileTrait.php [hunk 3]
```diff
@@ -149,11 +152,11 @@ public function fileCleanAll(AfterScenarioScope $scope): void {
    */
   public function fileDeleteManagedFiles(TableNode $nodesTable): void {
     $storage = \Drupal::entityTypeManager()->getStorage('file');
-    $filenames = $nodesTable->getColumn(0);
-    // Get rid of the column header.
-    array_shift($filenames);
-    foreach ($filenames as $filename) {
-      $ids = $this->fileLoadMultiple(['filename' => $filename]);
+    $fieldValues = $nodesTable->getColumn(0);
```
Comment (AlexSkrypnyk) [inline src/FileTrait.php:155]: do not use camelCase - for variables - please use snake_case

File: tests/behat/features/file.feature [hunk 4]
```diff
@@ -25,10 +25,10 @@ Feature: Check that FileTrait works for or D9
     And "example_image.png" file object exists
     And "example_audio.mp3" file object exists
     Given no managed files:
-      | path                 |
```
Comment (AlexSkrypnyk) [inline tests/behat/features/file.feature:28]: please add another test to show that a different field name also works.

**Review by AlexSkrypnyk — CHANGES_REQUESTED** [review]
@tannguyen04 
Please see my inline comments.

**Comment by tannguyen04** [comment]
Hi @AlexSkrypnyk 
I have updated as your request.

**Review by AlexSkrypnyk — APPROVED** [review]

**Pull request merged**